model/src/main/java/org/projectnessie/model/GetNamespacesResponse.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2022 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.model;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import java.util.List;
+import javax.validation.constraints.NotNull;
+import org.immutables.value.Value.Immutable;
+
+@Immutable
+@JsonSerialize(as = ImmutableGetNamespacesResponse.class)
+@JsonDeserialize(as = ImmutableGetNamespacesResponse.class)
+public interface GetNamespacesResponse {
```

Comment:
`EntriesResponse` is paginated... Should we extend `PaginatedResponse` too, even if we do not support pagination ATM?